File: DEV/BUT1/APL1.2/TP02/date.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>
int main(void) {
  struct date {
  int annee;
  int mois;
  int jour;
  };
  printf("La date du jour est : ");
  struct date tm ;
  struct tm *localtime(const long int* time_t );
  printf("%d ",tm.annee);
  printf("%d ",tm.mois);
  printf("%d ",tm.jour);
  printf("\n");
}

File: DEV/BUT1/APL2.1/TP05/date/Date.java
public class Date {
  // attribut
  private String date;
  public Date() {
    this.date = "0000-00-00";
  }
  // méthode
  // autre méthode
  //public String toString( String chaine) {
  //  return Integer.toBinaryString(chaine);
  //}
  public static String date(String annee, String mois, String jour){
    return annee+'-'+mois+'-'+jour;
    }
  public static void main(String[] args) {
    String annee=args[0];
    String mois=args[1];
    String jour=args[2];
    String dateFinale;
    dateFinale=date(annee,mois,jour);
    System.out.println(dateFinale);
  }
}